bigbluebutton-html5/imports/ui/components/chat/service.js

```diff
@@ -215,6 +224,17 @@ const updateUnreadMessage = (receiverID, timestamp) => {
   return UnreadMessages.update(receiverID, timestamp);
 };
 
+const createClosedChatSession = (chatID) => {
+
+  let currentClosedChats = Storage.getItem('closedChatList') || [];
+
+  if (!_.contains(currentClosedChats, chatID)) {
+    currentClosedChats.push(chatID);
+  }
+
+  Storage.setItem('closedChatList', currentClosedChats);
```

Comment:
Move `Storage.setItem('closedChatList', currentClosedChats);` inside the `if` block (above) please